src/agent/main.c

```diff
@@ -24,7 +24,7 @@ static const char *opt_config = NULL;
 // NOLINTEND(cppcoreguidelines-avoid-non-const-global-variables)
 
 static void usage(char *argv[]) {
-        fprintf(stderr, "Usage: %s [-H host] [-p port] [-c config] [-n name]\n", argv[0]);
+        hirte_log_errorf("Usage: %s [-H host] [-p port] [-c config] [-n name]\n", argv[0]);
```

Comment:
I don't think usage() should be using logging. This is a straight print.

Reply:
Yes, you are right. Changed/reverted it.